source/BankPanel.cpp

```diff
@@ -35,7 +35,20 @@ namespace {
 	// Column headings.
 	const string HEADING[6] = {"Type", "Principal", "Interest", "Term", "Payment", ""};
 	// X coordinates of the columns of the table.
-	const int COLUMN[5] = {-290, -180, -100, -30, 20};
+	struct Columns {
```

Comment:
I would like to add a method AddColumn(int x, DisplayText::Align align) as a simple version. This will be equivalent to AddColumn(x, {-1, align, Truncate::NONE}).